docs/zh/UserGuide/API/InfluxDB-Protocol.md

```diff
@@ -162,7 +175,7 @@ time                address name phone sex socre
    (1)插入第一条数据时，需要将新出现的三个 tag key 更新到 table 中，IoTDB 对应的记录 tag 顺序的 table 为：
 
    | database | measurement | tag_key | Order |
-         | -------- | ----------- | ------- | ----- |
+        | -------- | ----------- | ------- | ----- |
```

Comment:
It seems that this diff causes the form can not be displayed.

Reply:
I have solved this problem:D